### PR TITLE
Don't set juju snap channel and pip constraints

### PR DESCRIPTION
**Pull request by sabaini**
Use juju 2.9 in gate as we don't support newer Juju with Zaza yet

Depends-On: openstack-charmers/zaza#652

### CODE REVIEW

File: zuul.d/jobs.yaml [hunk 1]
```diff
@@ -530,8 +530,8 @@
       # constraints have pinning to install the right version of python-libjuju
       # based on what snap channel is being used to bootstrap the controller
       # from.
-      juju_snap_channel: '3.5/stable'
-      pip_constraints_url: 'https://raw.githubusercontent.com/openstack-charmers/zaza-openstack-tests/master/constraints/constraints-noble.txt'
+      # juju_snap_channel: '3.5/stable'
+      # pip_constraints_url: 'https://raw.githubusercontent.com/openstack-charmers/zaza-openstack-tests/master/constraints/constraints-noble.txt'
```
Comment (ajkavanagh) [inline zuul.d/jobs.yaml:534]: For noble, we're transitioning to juju 3.5/stable, and moving off from 2.9; thus we don't really want to revert this change.

**Review by ajkavanagh — CHANGES_REQUESTED** [review]
This is a revert effectively.

**Comment by ajkavanagh** [comment]
> Use juju 2.9 in gate as we don't support newer Juju with Zaza yet

> Depends-On: https://github.com/openstack-charmers/zaza/pull/652

We probably want to look at this again.  We need to move forward and use Juju 3.x for noble.

**Comment by sabaini** [comment]
Closing -- we're now running juju3 with zaza

**Pull request closed without merging**